azure-batch/azure/batch/batch_service_client.py

```diff
@@ -46,7 +46,7 @@ def __init__(
 
         super(BatchServiceClientConfiguration, self).__init__(base_url)
 
-        self.add_user_agent('batchserviceclient/{}'.format(VERSION))
+        self.add_user_agent('azure-batch/{}'.format(VERSION))
```

Comment:
@lmazuel @xingwu1 - did we change this or is this just a change from the generator?

Reply:
@lmazuel  use the readme to generate the code which has flag `--package azure-batch`. However, from earliest version batch sdk to now, we never use this flag to generate the sdk. I prefer the new agent name, but I am not sure whether this count breaking change or not.

Reply:
This is not breaking, but we can keep the old one for now for safety, no worries